fdbserver/tester.actor.cpp

```diff
@@ -913,7 +913,7 @@ ACTOR Future<Void> clearData(Database cx, Optional<TenantName> defaultTenant) {
 
 			std::vector<Future<Void>> deleteFutures;
 			for (auto const& [id, entry] : tenants.results) {
-				if (entry.tenantName != defaultTenant) {
+				if (entry.tenantName != defaultTenant.get()) {
```

Comment:
I don't think `defaultTenant` is guaranteed to be present:

```suggestion
				if (!defaultTenant.present() || entry.tenantName != defaultTenant.get()) {
```